source/reference/api/start.txt

```diff
@@ -151,12 +225,103 @@ Request
 Response
 ~~~~~~~~
 
+.. literalinclude:: /includes/api/responses/success.json
+   :language: json
+   :copyable: false
+
+Example 3 - Start Sync from Replica Set to Sharded Cluster
+----------------------------------------------------------
+
+Request
+~~~~~~~
+
+.. literalinclude:: /includes/api/requests/start-rs-shard.sh
+   :language: shell
+
+Response
+~~~~~~~~
+
 .. literalinclude:: /includes/api/responses/success.json
    :language: json
    :copyable: false
 
 Behavior
 --------
 
+State
+~~~~~
+
 If the ``start`` request is successful, ``mongosync`` enters the
 ``RUNNING`` state.
+
+Shard Collections
+~~~~~~~~~~~~~~~~~~~~~
+
+Synchronization from a replica set to a sharded cluster requires the 
+``sharding`` option, to specify how ``mongosync`` should shard collections.
```

Comment:
[suggestion]

IMO, breaking this into two separate sentences and using a similar second-person voice to the edit we made above would make this easier to understand. Consider something like:

"To synchronize a replica set to a sharded cluster, specify the ``sharding`` option. The ``sharding option`` specifies how ``mongosync`` shards collections."

Reply:
Fixed